### DISCUSSION TITLE
Log /metric and /healthz on errors only

### DISCUSSION THREAD

**Discussion by helmut72** [drakkan/sftpgo#1393] [Ideas]
Hi,

since some weeks I found a new playground and play with metrics/prometheus. Therefore I enable metrics on different server apps to see what can be useful or not. But some of the apps now "flood" their logs with metrics access logs, up to the seconds I have configured (eg. every 15secs).

On a homelab/low traffic system, 95% of the logs are now metric access logs. Nice to see always a 200, but it feels useless with time. Also another piece that force wear leveling on sd-cards (Raspberry).

Would be cool to have an option to log metrics if there are really errors. Same for /healthz for healthcheck in Docker - even if the frequency is lower there.

Thank you.

**Comment by drakkan** [comment]
https://github.com/drakkan/sftpgo/commit/397cad93dff57fbae59b08e6e46fac4ea60d9b94

**Reply by helmut72** [reply]
Sorry for late answer, was a busy week and used my spare time outside in this last hot days here. Have tried your edge image now and it works. Thank you very much!

Too bad I have few simple use cases only, eg. up/download my password file. Especially all this event features looks really interesting and would allow "SftpgoOps" instead of GitOps. But I have also gitea running.
